_overviews/core/value-classes.md

```diff
@@ -56,6 +56,46 @@ The following fragment of `RichInt` shows how it extends `Int` to allow the expr
 At runtime, this expression `3.toHexString` is optimised to the equivalent of a method call on a static object
 (`RichInt$.MODULE$.extension$toHexString(3)`), rather than a method call on a newly instantiated object.
 
+### Location of extension methods
+
+There are several options regarding where to place the implemented extension methods.
+They can be placed in a package object which makes them available within the package automatically.
+
+    package object mypackage {
+      implicit class RichInt(val self: Int) extends AnyVal {
+        def toHexString: String = java.lang.Integer.toHexString(self)
+      }
+    }
+
+If the extension methods shall be reused it might be feasible to implement them not implicitly and include them via traits.
+
+    // in file: src/main/scala/mypackage/mypackage.scala
```

Comment:
by convention this is `// in file: src/main/scala/mypackage/package.scala`

Reply:
Ups. Correcting.